modules/auto-react/events/messageCreate.js

```diff
@@ -26,6 +29,27 @@ async function checkMembers(msg) {
     });
 }
 
+/**
+ * Checks for member pings in a message and replys with the configured message
+ * @private
+ * @param msg
+ * @returns {Promise<void>}
+ */
+async function checkMembersReply(msg) {
+    const moduleConfig = msg.client.configurations['auto-react']['replys'];
```

Comment:
```suggestion
    const moduleConfig = msg.client.configurations['auto-react']['replies'];
```

This config file does not exist in your current project. Please commit.